lib/msal-browser/README.md

```diff
@@ -132,11 +142,9 @@ npm test
 npm run test:coverage
 ```
 
-## Implicit Flow vs Authorization Code Flow with PKCE
-
-`@azure/msal-browser` implements the [OAuth 2.0 Authorization Code Flow with PKCE](https://tools.ietf.org/html/rfc7636) for browser-based applications. This is a significant improvement over the Implicit Flow that was used in `@azure/msal`, `msal` or `adal-angular`.
+## Authorization Code Flow with Proof Key for Code Exchange
```

Comment:
Add the acronym (PKCE) to the end of this line:  Authorization Code Flow with Proof Key for Code Exchange (PKCE)